utils/src/bench.rs

```diff
@@ -194,7 +194,7 @@ macro_rules! ncl_bench_group {
                                 .unwrap()
                                 .transformed_term;
                             if bench.eval_mode == $crate::bench::EvalMode::TypeCheck {
-                                cache.parse(id).unwrap();
+                                cache.parse(id, InputFormat::Nickel).unwrap();
```

Comment:
I think you should either:

1. Use a fully qualified name for `InputFormat`, that is `nickel_lang_core::cache::InputFormat`
2. Import it a few line above, inside the macro, and keep it as `InputFormat`

But in any case, the consumers/callers of `ncl_bench_group` should have to import it themselves (as you had to do in various bench files). This leads to code duplication, but more importantly, callers of `nlc_bench_group` have to magically guess that they need to do this import (or wait for a compilation error), while the macro should - and can - be self-contained.

Reply:
Edit: I meant should **NOT** have to import it themselves.

Reply:
Yep, right now I see. I missed that the code I edited was inside a macro, sure it's a mistake. So, I'll import it in macro, and get rid of those imports from benches:)